cognite/client/_api/datapoints.py

```diff
@@ -1497,15 +1560,14 @@ def insert_dataframe(self, df: pd.DataFrame, external_id_headers: bool = True, d
 
         Warning:
             You can not insert datapoints with status codes using this method (``insert_dataframe``), you'll need
-            to use the :py:meth:`~DatapointsAPI.insert` method instead (once support is added in an upcoming release).
+            to use the :py:meth:`~DatapointsAPI.insert` method instead (or :py:meth:`~DatapointsAPI.insert_multiple`)!
```

Comment:
Probably a personal preference, but feels more serious
```suggestion
            to use the `~DatapointsAPI.insert` method instead (or `~DatapointsAPI.insert_multiple`)!
```

Reply:
This removes the clickable links to the other SDK functions when I build the docs locally. How/what change does this cause on your machine?

Without ~~:py:meth:~~
<img width="681" alt="image" src="https://github.com/cognitedata/cognite-sdk-python/assets/8521241/cd14240a-a006-4c88-9ae4-351e94334059">

With :py:meth:
<img width="692" alt="Screenshot 2024-04-15 at 23 40 09" src="https://github.com/cognitedata/cognite-sdk-python/assets/8521241/c230819c-5fc5-44ea-a36c-171a0a31299d">

Reply:
Or do you mean just without the `()`?

Reply:
Oh, nvm, they are inserted automatically by sphinx